benchmarks/templates/benchmarks/central_profile.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'benchmarks/base.html' %}
+{% load static %}
+{% block content %}
+
+    {% if user %}
+        <section id="profile" class="container center">
+            <section id = profile-info>
+                <h1>{{ user.email }}</h1>
+
+                <form action="{% url 'display-name' %}" method="post">
+                    {% csrf_token %}
+                    <label class="label">Display name</label>
+                    <div class="field has-addons">
+                        <p class="control">
+                            <input class="input" type="text" name="display_name" value="{{ user.display_name }}">
+                        </p>
+                        <p class="control">
+                            <input type="submit" value="Save" class="button">
+                        </p>
+                    </div>
+                </form>
+            </section>
+
+
+
+            <div id="domain_overviews" class="box">
+                <div id="vision_overview" class="box">
+                    <div>
+                        View your <strong>vision</strong> plugins and leaderboard
+                        <a href="http://{{ request.get_host }}/profile/vision">here</a>.
+
+                        Submit new vision plugins <a href="{{ request.build_absolute_uri }}/">here</a>.
```

Comment:
`request.build_absolute_uri` looks incorrect here, it's the same below for language.